docs/content/guide/forms.ngdoc

```diff
@@ -488,7 +488,7 @@ The following example shows how to add two-way data-binding to contentEditable e
   </file>
 
   <file name="script.js">
-    angular.module('form-example2', []).directive('contenteditable', function() {
+    angular.module('form-example2', []).directive('contentEditable', function() {
```

Comment:
This would look for a `content-editable` attribute (or element), not `contenteditable`.
(Note that HTML is case-insensitive.)